lib/Extension/LanguageServerSymbolProvider/Tests/Unit/Adapter/TolerantDocumentSymbolProviderTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Phpactor\Extension\LanguageServerSymbolProvider\Tests\Unit\Adapter;
+
+use Microsoft\PhpParser\Parser;
+use PHPUnit\Framework\TestCase;
+use Phpactor\Extension\LanguageServerSymbolProvider\Adapter\TolerantDocumentSymbolProvider;
+
+class TolerantDocumentSymbolProviderTest extends TestCase
+{
+    public function testPropertiesInTraits(): void
+    {
+        $provider = new TolerantDocumentSymbolProvider(new Parser);
```

Comment:
all `new` classes should have the parenthesis (e.g. `new Parser()`) maybe add the `new_with_parentheses` rule to `.php-cs-fixer.dist.php`? (https://mlocati.github.io/php-cs-fixer-configurator/#version:3.40%7Cfixer:new_with_parentheses)

Reply:
Do you mind I add the rule to the `.php-cs-fixer.dist.php` along with the PR? Or just only fix the parenthesis?

Reply:
happy if you did it in this PR, but your call :slightly_smiling_face:

Reply:
I extracted it to a dedicated PR to let you smoothly review the file changes. #2447